ci/conda/recipes/libmrc/conda_build_config.yaml

```
@@ -26,8 +26,8 @@ cuda_compiler_version:
   - 11.8
 
 python:
-  - 3.8
   - 3.9
+  - 3.10
```

```suggestion
  - 3.10
```
We were optimistic that we would support multiple versions of python at first which would make this build matrix necessary. However, in `run_conda_build.sh` we just ignore this variant 100% of the time so we should just remove this.